### ISSUE TITLE
Implement SparkLabelEncoder

### ISSUE THREAD

**Issue by kszucs** [lensacom/sparkit-learn#25]
see https://github.com/scikit-learn/scikit-learn/blob/master/sklearn/preprocessing/label.py#L56
